src/__snapshots__/browserslist-config-canopy.test.js.snap

```diff
@@ -0,0 +1,35 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`browserslist-config-canopy exports a valid browserslist config that is what we want for Canopy 1`] = `
+Array [
+  "chrome 64",
+  "chrome 63",
+  "chrome 62",
+  "chrome 61",
+  "chrome 60",
+  "chrome 59",
+  "chrome 58",
+  "chrome 57",
+  "chrome 56",
+  "chrome 55",
```

Comment:
a thought I had: this snapshot only gets updated when a new commit happens, right? but what is actually given when extending this config may be different than the snapshot, right?

e.g. this snapshot has chrome 64-55 in it, in a couple of months the latest 10 will be versions 66-57 -- and that means that this snapshot could potentially be slightly misleading, right?

Reply:
yeah, if people look to this to see what browsers are supported. I'm thinking that probably no one will ever look at this snapshot file though unless they need to update it. Do you think it should be removed because it's more confusing than it's worth?

Reply:
Eh, I dunno. I like the idea, but dislike the potential for confusion. In the end, though, I think you're right that it probably won't get looked at anyway except for when new commits happen (which are probably going to be pretty infrequent). 

I'll let you decide.

Reply:
I think the use I see in it is that it forces someone to actually look at a list of browsers. Without this test, you would have to manually do `npx browserslist` in order to see if your new config does what you want it to. But this snapshot test forces your hand and makes you look at it if it changes.